packages/client/src/app/pages/news/news.component.html

```diff
@@ -1,12 +1,39 @@
-<div class="news-container">
-    <div class="news-header">
-        <h2>News</h2>
+<div class="tools">
+    <div class="tool-buttons">
+        <button matRipple class="tool"><i-feather name="filter"></i-feather>Filter</button>
     </div>
-    <hr class="header-divider">
 </div>
 
-<div style="text-align: center; margin-top: 2rem;">
-    <h3 style="margin-bottom: 0;">Pardon the dust!</h3>
-    <p style="margin-top: 0; margin-bottom: 0; font-style: italic;">Looks like you came here a bit too early.</p>
-    <p style="margin-top: 0; font-style: italic;">Keep an eye on the patch notes to find out when this feature will be available!</p>
-</div>
\ No newline at end of file
+<ng-container *ngIf="posts.totalDocs === 0; else hasDocs">
+    <div style="text-align: center; margin-top: 2rem;">
+        <h3 style="margin-bottom: 0;">Surprisingly, there's nothing here.</h3>
+        <p style="margin-top: 0; margin-bottom: 0; font-style: italic;">Contact an admin if you think something should be here.</p>
```

Comment:
Would these styles be better off as classes? I feel like these get reused in a few places.